src/app/app.component.scss

```diff
@@ -1,3 +1,6 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:300);
```

Comment:
general stuff like those import and the body style should be placed in the file src/styles.scss